<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MGridInfoWidget</class>
 <widget class="QWidget" name="MGridInfoWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Index Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_col">
     <item>
      <widget class="QLabel" name="label_col">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Column Counts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_col"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_row">
     <item>
      <widget class="QLabel" name="label_row">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Row Counts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_row"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MGridInfoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
